engine/runtime/src/main/scala/org/enso/compiler/pass/analyse/AliasAnalysis.scala

```diff
@@ -136,10 +136,23 @@ case object AliasAnalysis extends IRPass {
     val topLevelGraph = new Graph
 
     ir match {
-      case _: Method.Conversion =>
-        throw new CompilerError("Conversion methods are not yet supported.")
-      case m @ IR.Module.Scope.Definition.Method
-            .Explicit(_, body, _, _, _) =>
+      case m: Method.Conversion =>
```

Comment:
Something strange with the name qualification here. Why is `Method` imported for this, but then used fully qualified in the next case?